lib/XTR/ImageProcessingTask.php

```diff
@@ -97,7 +97,9 @@ private function runImageProcessing()
             $count++;
 
             // Release image and save
-            $media->release(md5_file($filePath), filesize($filePath));
+            list($updatedImgWidth, $updatedImgHeight) = @getimagesize($filePath);
```

Comment:
Have we explored if there is a more efficient way to do this? For example, does the image processing service know how it has resized the image? Could it return the new width/height for us to use?

Reply:
Sure, but can we avoid calling `getimagesize` twice? If the resizeImage function knows what it resized it to, could that be returned instead?

Reply:
Hi again, Dan! I've made a few changes and would like to request a re-review, please. Thanks again :)